app/constants/config.js

```diff
@@ -1,7 +1,7 @@
 require('dotenv').config();
 
 const {
-  API_URL = 'https://maap-stg.instedd.org',
+  API_URL = 'http://localhost:3000',
```

Comment:
I'm not against this, just checking that you intended this change to be here

Reply:
You can edit this on a .env file in your project root folder

Currently this is the url that is used on the build that is released, so we shouldn't change this.

There's a pending task (in my mind, but it should be on an issue) about making the build api url dependent on environment variables during ci build time (for example, for production branch we will need to edit this file for now, but it should be just an environment variable)